lib/monitoring/aws-rds/AuroraClusterMonitoring.ts

```diff
@@ -0,0 +1,239 @@
+    this.serverlessDatabaseCapacityMetric =
+      metricFactory.metricServerlessDatabaseCapacity();
+    this.cpuUsageMetric = metricFactory.metricAverageCpuUsageInPercent();
+    this.selectLatencyMetric = metricFactory.metricSelectLatencyP90InMillis();
+    this.insertLatencyMetric = metricFactory.metricInsertLatencyP90InMillis();
+    this.updateLatencyMetric = metricFactory.metricUpdateLatencyP90InMillis();
+    this.deleteLatencyMetric = metricFactory.metricDeleteLatencyP90InMillis();
+    this.commitLatencyMetric = metricFactory.metricCommitLatencyP90InMillis();
+
+    const namingStrategy = new MonitoringNamingStrategy({
+      ...props,
+      fallbackConstructName: metricFactory.clusterIdentifier,
+      namedConstruct: props.cluster,
+    });
+    this.title = namingStrategy.resolveHumanReadableName();
+    this.url = scope
+      .createAwsConsoleUrlFactory()
+      .getRdsClusterUrl(metricFactory.clusterIdentifier);
+    const alarmFactory = this.createAlarmFactory(
+      namingStrategy.resolveAlarmFriendlyName()
+    );
+    this.usageAlarmFactory = new UsageAlarmFactory(alarmFactory);
+    this.connectionAlarmFactory = new ConnectionAlarmFactory(alarmFactory);
+
+    this.usageAnnotations = [];
+    this.connectionAnnotations = [];
+
+    for (const disambiguator in props.addCpuUsageAlarm) {
+      const alarmProps = props.addCpuUsageAlarm[disambiguator];
+      const createdAlarm = this.usageAlarmFactory.addMaxCpuUsagePercentAlarm(
+        this.cpuUsageMetric,
+        alarmProps,
+        disambiguator
+      );
+      this.usageAnnotations.push(createdAlarm.annotation);
+      this.addAlarm(createdAlarm);
+    }
+
+    for (const disambiguator in props.addMinConnectionCountAlarm) {
+      const alarmProps = props.addMinConnectionCountAlarm[disambiguator];
+      const createdAlarm =
+        this.connectionAlarmFactory.addMinConnectionCountAlarm(
+          this.connectionsMetric,
+          alarmProps,
+          disambiguator
+        );
+      this.connectionAnnotations.push(createdAlarm.annotation);
+      this.addAlarm(createdAlarm);
+    }
+
+    for (const disambiguator in props.addMaxConnectionCountAlarm) {
+      const alarmProps = props.addMaxConnectionCountAlarm[disambiguator];
+      const createdAlarm =
+        this.connectionAlarmFactory.addMaxConnectionCountAlarm(
+          this.connectionsMetric,
+          alarmProps,
+          disambiguator
+        );
+      this.connectionAnnotations.push(createdAlarm.annotation);
+      this.addAlarm(createdAlarm);
+    }
+
+    for (const disambiguator in props.addMaxServerlessDatabaseCapacityAlarm) {
+      const alarmProps =
+        props.addMaxServerlessDatabaseCapacityAlarm[disambiguator];
+      const createdAlarm = this.usageAlarmFactory.addMaxUsageCountAlarm(
+        this.serverlessDatabaseCapacityMetric,
+        alarmProps,
+        disambiguator
+      );
+      this.connectionAnnotations.push(createdAlarm.annotation);
```

Comment:
Seems like this should be using another array that's used for `createServerlessDatabaseCapacityWidget` rather than putting it with the connections.